src/test/rustdoc-gui/search-input-mobile.goml

```diff
@@ -0,0 +1,14 @@
+// Test to ensure that you can click on the search input, whatever the width.
+// The PR which fixed it is: https://github.com/rust-lang/rust/pull/81592
+goto: file://|DOC_PATH|/index.html
+size: (463, 700)
+// We first check that the search input isn't already focused.
+fail: true
```

Comment:
What does this do?

Reply:
I use it to reverse the `assert` checks. So once we set `fail` to true, if any following command succeeds, the script errors. Documentation here: https://github.com/GuillaumeGomez/browser-UI-test/#fail

Reply:
That seems confusing. Rather than having it as state, could you add a `not` check? So
```
fail: true
assert: ("input.search-input:focus")
fail: false
```
would turn into something like
```
assert: not ("input.search-input:focus")
```